dbt_copilot_helper/commands/codebase.py

```diff
@@ -74,3 +78,45 @@ def prepare():
         phase_contents = templates.get_template(f".copilot/phases/{phase}.sh").render()
 
         click.echo(mkfile(Path("./.copilot"), f"phases/{phase}.sh", phase_contents, overwrite=True))
+
+
+@codebase.command()
+@click.option("--app", help="AWS application name", required=True)
+@click.option("--codebase", help="GitHub codebase name", required=True)
+@click.option("--commit", help="GitHub commit hash", required=True)
+def build(app, codebase, commit):
+    """Trigger a CodePipeline pipeline based build."""
+    try:
+        load_application(app)
+    except ApplicationNotFoundError:
+        click.secho(
+            f"""The account "{os.environ.get("AWS_PROFILE")}" does not contain the application "{app}"; ensure you have set the environment variable "AWS_PROFILE" correctly.""",
+            fg="red",
+        )
+        raise click.Abort
+
+    check_if_commit_exists = subprocess.run(
+        ["git", "branch", "-r", "--contains", f"{commit}"], capture_output=True, text=True
+    )
+
+    if check_if_commit_exists.stderr:
+        click.secho(
+            f"""The commit hash "{commit}" either does not exist or you need to run `git pull`.""",
+            fg="red",
+        )
+        raise click.Abort
+
+    if click.confirm(
+        f"""You are about to build "{app}" for "{codebase}" with commit "{commit}". Do you want to continue?"""
+    ):
+        boto3.client("codebuild").start_build(
+            projectName=f"codebuild-{app}-{codebase}",
+            artifactsOverride={"type": "NO_ARTIFACTS"},
+            sourceVersion=commit,
+        )
+
+        return click.echo(
+            "Your build has been triggered and you can check your build progress in the AWS Console."
```

Comment:
May read better as: "Your build has been triggered. Check your build progress in the AWS Console."

Reply:
Another option could be to link to the build, though the user would also need to be prompted to log into the AWS account. See the `get_build_url` function in [ci-image-builder](https://github.com/uktrade/ci-image-builder/blob/main/image_builder/notify.py#L118)

Reply:
Went with Ant's suggested change for now! Resolved in https://github.com/uktrade/copilot-tools/pull/255/commits/28d645a856e80cb8ea9ec1c977a4561062fda34b.